openhands/agenthub/codeact_agent/function_calling.py

```diff
@@ -506,7 +507,17 @@ def response_to_actions(response: ModelResponse) -> list[Action]:
                 logger.debug(
                     f'TOOL CALL: str_replace_editor -> file_editor with code: {code}'
                 )
-                action = IPythonRunCellAction(code=code, include_extra=False)
+
+                if arguments['command'] == 'view':
+                    # TODO: convert to FileReadAction?
+                    action = IPythonRunCellAction(code=code, include_extra=False)
```

Comment:
Can we do this? We can just modify on the runtime side about how we handle the file readt

Reply:
Yep, I can have a look